src/Persistence/Sql.php

```diff
@@ -592,6 +592,13 @@ public function action(Model $model, string $type, array $args = [])
                 $this->initQueryConditions($model, $query);
                 $model->hook(self::HOOK_INIT_SELECT_QUERY, [$query, $type]);
 
+                // MSSQL does not support EXISTS everywhere, so wrap in SELECT
+                if ($this->connection instanceof \atk4\dsql\Mssql\Connection) {
```

Comment:
@georgehristov this will return an Expression instead of Query now, how to modify it?